Sources/Player/Common/Event/NotificationsHandler.swift

```diff
@@ -58,4 +60,40 @@ final class NotificationsHandler {
 			self.listener.djSessionTransitioned(to: transition)
 		}
 	}
+
+	func offliningStarted(for mediaProduct: MediaProduct) {
+		queue.async {
+			self.offlineEngineListener?.offliningStarted(for: mediaProduct)
+		}
+	}
+
+	func offliningProgress(for mediaProduct: MediaProduct, is downloadPercentage: Double) {
```

Comment:
[suggestion] maybe for consistency with the other names where we use "started", "completed", etc.
```suggestion
	func offliningProgressed(for mediaProduct: MediaProduct, is downloadPercentage: Double) {
```

Reply:
Great suggestion